packages/react-core/src/components/TimePicker/TimePicker.tsx

```diff
@@ -233,6 +258,10 @@ export class TimePicker extends React.Component<TimePickerProps, TimePickerState
     if (splitTime.length < 2) {
       time = `${time}${delimiter}00`;
       splitTime = time.split(delimiter);
+      // build time without seconds when includeSeconds is true
```

Comment:
Did you mean to write when includeSeconds is *false?

Reply:
So there was an issue when seconds is included in the input and a user would try navigating to the menu. What _should_ happen is they're brought to the exact or the closest time (2:30:00 in the input should bring them to 2:30 in the menu of options). But since the contents of the menu doesn't include seconds, the user would always be brought to the first item in the list (12:00 AM).

Building that temporary time _without_ the seconds, even when they're included as a prop, basically just makes the scrolling work as intended. I will update the comment to reflect the reasoning for building the time without seconds, even when the includeSeconds is true.

Reply:
Makes sense. Thanks for the explanation!